syft/formats/common/spdxhelpers/to_format_model.go

```diff
@@ -27,12 +27,30 @@ const (
 	noAssertion = "NOASSERTION"
 )
 
+var DocumentDescribesRelationship = &spdx.Relationship{
```

Comment:
since this isn't being used elsewhere we shouldn't be exporting it. (it could probably be inlined within `ToFormatModel`)